### ISSUE TITLE
[BUG] Hover text date is wrong when lc is folded

### ISSUE THREAD

**Issue by ignacioreyes** [alercebroker/ztf_explorer#351]
**Describe the bug**
When looking at a folded lightcurve, the hover text info about the date of the alert is wrong. It seems that the phase is being used as a date.

**To Reproduce**
1. Look at any object whose lightcurve can be folded.
2. Fold the lightcurve.
3. Place the cursor over any detection.
4. Look at the date info. You will see dates from the year 1858.

**Expected behavior**
It should show the proper date or no date at all.

**Screenshots**
![image](https://user-images.githubusercontent.com/13869886/135185013-c2b7a1d8-4cf9-4651-bb13-8c51d59a9430.png)

**Desktop (please complete the following information):**
 - Mozilla Firefox
 - Version 92.0

**Comment by dirodriguezm** [comment]
You're right. It is taking the phase as it was MJD. I guess the easiest would be to not show dates on folded plot.
